decidim-core/lib/decidim/core/test.rb

```diff
@@ -76,3 +76,6 @@
 require "decidim/core/test/shared_examples/resource_endorsed_event_examples"
 require "decidim/core/test/shared_examples/versions_controller_examples"
 require "decidim/core/test/shared_examples/mcell_examples"
+
+require "#{Gem.loaded_specs["decidim-core"].full_gem_path}/app/jobs/decidim/application_job"
+require "#{Gem.loaded_specs["decidim-core"].full_gem_path}/app/jobs/decidim/hide_all_created_by_author_job"
```

Comment:
I understand that this is necessary because we're adding the `Decidim::DummyResources::HideAllCreatedByAuthorJob`. in `decidim-dev/lib/decidim/dev/test/rspec_support/component.rb`. I think we should rethink how to handle this as its ugly, but I don't have any proposal for now 😞

Reply:
Created #10680 to address this issue.